build.py

```diff
@@ -462,6 +462,24 @@ def dali_cmake_args():
         '-DTRITON_DALI_SKIP_DOWNLOAD=OFF',
     ]
 
+def install_dcgm_libraries():
+    return '''
+# Install DCGM
+RUN apt-get update && apt-get install -y --no-install-recommends software-properties-common
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin \
+&& mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600 \
+&& apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/7fa2af80.pub \
+&& add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /"
+RUN apt-get update \
+&& apt-get install -y datacenter-gpu-manager
+'''
+
+def fail_if_dcgm_api_export_exists():
+    return '''
+# Note: The current version is missing header files so we applied a dcgm_api_export.h to fix the build. This fix should be removed
+# after DCGM team fixes this issue in their new release.
+RUN if test -s /usr/include/dcgm_api_export.h;then echo "/usr/include/dcgm_api_export.h exists! Remove copy from build.py" | exit 1;fi
```

Comment:
Replace '|' with ';'? Why are you piping the echo to "exit 1". Did you test this to make sure the build fails as expected.

Reply:
I did. Figured out a way to make a patch so removing all of this anyway